src/pages/Dashboard/Modals/SupplyWithdraw/SupplyWithdrawForm.tsx

```diff
@@ -105,7 +134,7 @@ export const SupplyWithdrawContent: React.FC<
           components={{
             White: <Typography component="span" variant="small1" css={styles.whiteLabel} />,
           }}
-          values={{ amount: format(maxInput), symbol: assetId?.toUpperCase() }}
+          values={{ amount: format(maxInput.minus(amount)), symbol: assetId?.toUpperCase() }}
```

Comment:
This value shouldn't scale with `amount`, since it doesn't represent an hypothetical value but a value that's currently accurate. Otherwise it will tell the users their max withdraw amount or wallet balance is different from what it actually is.